### DISCUSSION TITLE
problem due to cloudflare

### DISCUSSION THREAD

**Discussion by Kirito2677** [community/community#19088] [Other Feature Feedback, Questions, & Ideas]
Hello, I have a problem with the tachiyomi app, which cannot read some manga sites because of cloudflare, is there a solution?

**Comment by mohitkyadav** [comment]
Hey @Kirito2677 if you think it's a bug, you might want to open an issue for that [here](https://github.com/tachiyomiorg/tachiyomi/issues).

**Comment by theteladras** [comment]
@Kirito2677 This might not be the relevant place for what you are asking.
